aiohttp/web_log.py

```diff
@@ -138,8 +138,12 @@ def _format_o(
     def _format_a(request: BaseRequest, response: StreamResponse, time: float) -> str:
         if request is None:
             return "-"
-        ip = request.remote
-        return ip if ip is not None else "-"
+        ip = request.headers.get("X-FORWARDED-FOR", None)
```

Comment:
This header must only be used if you trust the service you are connected to. This is a security vulnerability in it's current state and can't be merged.

If we wanted something like this, it'd need to be user configurable or something. Maybe better to open an issue for this one (if none already exists) and discuss the solution first.